### ISSUE TITLE
Paragraphs and quotes

### ISSUE THREAD

**Issue by vr8hub** [standardebooks/jonathan-swift_gullivers-travels#6]
I was going to open a PR on what I believed to be a mis-placed quote mark, when I discovered that the scans don't have quotes at all. I reviewed the history in Google groups to see if something had been done along the lines of _Gil Blas_, but nothing is mentioned there. The producer did mention a possibly newer scan that had more paragraph breaks but that wasn't dated, and you asked him to find one with a date, but nothing more was said, and the scan that is in the metadata has neither the quotes nor the more frequent paragraph breaks.

The transcription has both, so theoretically an edition exists with them that PG approved, but we don't have those scans, and I've looked at a dozen or so on IA, and I don't find any, either.

Unlike Gil Blas, this doesn't sound to me like dialog, it sounds to me like a _description_ of dialog. In short, I don't the quotes belong at all. But even if you decide they do, I think they're wrong in places.

So, before I opened a PR for the one I found (while I was finding exclusions for one of the lint tests), I wanted to alert you to the situation.

**Comment by acabal** [comment]
Yes I think you're right, there must be some edition floating around out there with quotation marks that PG used, because that's a very early ebook so unlikely that they did anything more than a straight transcription.

I also agree that the quotes don't make sense. It might not be that hard to remove them. If we can find a transcription of an edition without quotes, searching it for any quotation marks would reveal the few cases where they legitimately exist. Then we can remove the rest. I don't have time to do that now though!

**Comment by vr8hub** [comment]
No, no, I didn't expect you did. :) I just wasn't sure whether what I was thinking was correct. I just wanted to make sure I wasn't missing something; I'll look into it further.
